### PR TITLE
[vanguard] Remove code about cluster

### PR DESCRIPTION
**Pull request by peoplepeople000**
# Description
Removed most of the code that contain cluster, but some code that don't affect the running of the app(ex. .json) or relating to firebase haven't been removed

# Changes
Now there's no cluster.

# Notes

# Checklist

- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] Any changes to strings have been published to our translation tool
- [x] I conducted basic QA to assure all features are working
- [x] I requested code review from other team members

# Resolved Issues

### CODE REVIEW

File: src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx [hunk 1]
```diff
@@ -161,22 +150,18 @@ const TargetCategorySelect: React.FC<{
...
   const dispatch = useDispatch();
   const reportData = useSelector((state: IRootState) => state.report.data);
   const targetName = reportData.target.name;
-
-  let targetNameOptions: string[]; // decided by floor and targetCategory
+  const targetNameWithCategory = targetNameSelections;
+  let targetNameOptions: string[]; // decided by targetCategory
 
   if (reportData.floor && reportData.target.category) {
-    targetNameOptions = getOptions(cluster?.data.name || "").targetName.find(
```
Comment (linyc0817) [inline src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx:176]: Seems like `getOptions`  is not  used now, remember to remove it.

File: src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx [hunk 2]
```diff
@@ -25,13 +22,17 @@ import poiAddDrawerTargetCategory from "../../../assets/images/poiAddDrawerTarge
...
+import targetNameSelections from "../../../assets/data/options/selections/target-name.json";
+
+const FloorSelect: React.FC = () => {
   const { t } = useTranslation();
 
   const dispatch = useDispatch();
   const reportData = useSelector((state: IRootState) => state.report.data);
   const floor = reportData.floor;
-  const floorOptions = getOptions(cluster?.data.name || "").floor;
+  const floorOptions = floorSelctions;
```
Comment (linyc0817) [inline src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx]: Why need to assign the data again here? Maybe use `floorSelectoins` directly is better. Same as `targetCategorySelections` & `targetNameSelections`.

**Review by linyc0817 — CHANGES_REQUESTED** [review]
Overall looks good to me.
Some little suggestions!

**Comment by linyc0817** [comment]
Just keep the `cluster` in api/store! No need to remove.

File: src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx (hunk 1 above)
Comment (peoplepeople000) [reply src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx:176]: Ok, I remove the createOptions.ts file

File: src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx (hunk 2 above)
Comment (peoplepeople000) [reply src/components/Drawer/AddReportDrawer/AddReportDrawerContent.tsx]: Ok, I modified the code.

**Review by linyc0817 — APPROVED** [review]
Well done.

**Pull request merged**